db/migrate/20170727144624_add_import_store_to_tables.rb

```diff
@@ -0,0 +1,7 @@
+class AddImportStoreToTables < ActiveRecord::Migration[5.1]
+  def change
+    add_column :clients, :access_import, :jsonb
+    add_column :volunteers, :access_import, :jsonb
+    add_column :journals, :access_import, :jsonb
```

Comment:
And doesn't this even require creation of an extra model?
Plus the columns can be dropped again, after import is run for the very last time.